.travis.yml

```diff
@@ -20,6 +20,7 @@ env:
     - TEST_SPECIFIC_MODULES=presto-hive TEST_FLAGS="-P test-hive-materialized"
     - TEST_SPECIFIC_MODULES=presto-hive TEST_FLAGS="-P test-hive-recoverable-grouped-execution"
     - TEST_SPECIFIC_MODULES=presto-hive TEST_FLAGS="-P test-hive-pushdown-filter-queries"
+    - TEST_SPECIFIC_MODULES=presto-hive TEST_FLAGS="-P test-hive-selective-distributed-queries"
```

Comment:
@bhhari 

- Let's not create a separate Travis job, but rather re-use test-hive-pushdown-filter-queries one.
- I'd name the new test TestHivePushdownFilterQueries, but we already have a test with that name. One option is to just modify the existing test to extend `AbstractTestDistributedQueries`. Another option is to name the new test "basic" and have the existing test be the "advanced" addition to it. This would give us TestHivePushdownFilterQueriesBasic and TestHivePushdownFilterQueries. 
- The new test should enable both filter pushdown and subfield pruning.
- We also need a pushdown-filter version of the TestHiveIntegrationSmokeTest, but it can be added in a separate PR.

I'm going to look into HivePartitionManager.java change next.